_sass/_variables.scss

```diff
@@ -36,3 +36,4 @@ $black-color: #000000 !default;
 $code-bg-color-light:     rgba($purple-color, 0.05);
 $code-bg-color-dark:      #2c3237 !default;
 
+$dark-bg-color: #1C1C1D;
```

Comment:
minor: could you please indent this the same way as `$code-bg-color-*` variables and add a new line at the end of the file?

Reply:
OK. The variable `$dark-bg-color` is supposed to be named with `$grey-color-dark`. I will make changes later.